<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmember</class>
 <widget class="QDialog" name="addmember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSaveMember">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-----QPushButton-----*/
QPushButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(68, 188, 128, 255), stop:1 rgba(76, 209, 143, 255));
color: black;
font: 8pt &quot;Century Gothic&quot;;
min-width: 80px;
border-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: rgb(38, 156, 200);
padding: 5px;}

QPushButton::flat
{
background-color: transparent;
border: none;
color: #fff;}

QPushButton::disabled
{
background-color: rgb(131, 165, 174);
color: black;
border-color: black;}

QPushButton::hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(54, 149, 101, 255), stop:1 rgba(64, 175, 119, 255));
border: 1px solid rgb(41, 170, 182);}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QFrame" name="addBookTabFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 58, 68);
border-radius: 16px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="addBookTtle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>ADD MEMBER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="AddMemberFormFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 52, 61);
border-radius: 16px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="txtMemberName">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border: 2px solid rgb(96, 101, 124);
border-radius: 15px;
color: rgb(255, 255, 255);
font: 12pt &quot;Century Gothic&quot;;
padding-left:20px;
padding-right: 20px;
background-color: rgb(95, 95, 112);
}


QLineEdit::hover{
border: 2px solid rgb(43, 153, 213);
}

QLineEdit::focus{
border: 2px solid rgb(170, 255, 255);
background-color:rgb(43,45,56);
}</string>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtMemberPhone">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border: 2px solid rgb(96, 101, 124);
border-radius: 15px;
color: rgb(255, 255, 255);
font: 12pt &quot;Century Gothic&quot;;
padding-left:20px;
padding-right: 20px;
background-color: rgb(95, 95, 112);
}


QLineEdit::hover{
border: 2px solid rgb(43, 153, 213);
}

QLineEdit::focus{
border: 2px solid rgb(170, 255, 255);
background-color:rgb(43,45,56);
}</string>
       </property>
       <property name="placeholderText">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtMemberEmail">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border: 2px solid rgb(96, 101, 124);
border-radius: 15px;
color: rgb(255, 255, 255);
font: 12pt &quot;Century Gothic&quot;;
padding-left:20px;
padding-right: 20px;
background-color: rgb(95, 95, 112);
}


QLineEdit::hover{
border: 2px solid rgb(43, 153, 213);
}

QLineEdit::focus{
border: 2px solid rgb(170, 255, 255);
background-color:rgb(43,45,56);
}</string>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>addBookTabFrame</zorder>
  <zorder>btnSaveMember</zorder>
  <zorder>AddMemberFormFrame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
